src/models/AppState.js

```diff
@@ -481,6 +481,15 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
         fn: function (policy) {
           return policy && policy.rememberDeviceByDefault;
         }
+      },
+      'unauthenticatedToken': {
+        deps: ['transaction'],
+        fn: function (transaction) {
+          if (transaction && transaction.status === 'UNAUTHENTICATED' && transaction.data) {
```

Comment:
The flow is for login into an app that uses a custom login url (this is configured in the App settings, general tab). 
So the user goes to the deep link for the app login (something like http://org.com/home/salesforce/{sha}) and then we redirect to the custom login, but the url at this times comes with a stateToken. So we bootstrap the widget with such stateToken, and in the first call to the authn api we get back an UNAUTHENTICATED state.
I think we could do that with the AuthJs if we want to keep the stateToken out of the signinwidget world. As of now, my understanding is that for primaryAuth our AuthJs does not include the stateToken. We had a meeting with Karl and Len and all the action items were on the signin widget side.
@lboyette-okta ^